src/main/scala/beam/agentsim/agents/ridehail/RideHailAgent.scala

```diff
@@ -561,6 +568,7 @@ class RideHailAgent(
           currentBeamVehicle.useParkingStall(stall)
           parkAndStartRefueling(stall)
           isOnWayToParkAtStall = None
+          eventsManager.processEvent(new PersonLeavesVehicleEvent(latestObservedTick, Id.createPersonId(id), vehicle.id))
```

Comment:
The old code had this in one place, but it seems to have split into two - so I added both - fix if needed @colinsheppard